src/features/user/Settings/ApiKey/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { useTranslation } from 'next-i18next';
+import { ReactElement } from 'react';
+import CenteredContainer from '../../../common/Layout/CenteredContainer';
+import DashboardView from '../../../common/Layout/DashboardView';
+import SingleColumnView from '../../../common/Layout/SingleColumnView';
+import ApiKeyForm from './ApiKeyForm';
+
+export default function ApiKey(): ReactElement | null {
+  const { t, ready } = useTranslation('me');
+
+  return ready ? (
+    <DashboardView
+      title={t('apiKey')}
+      subtitle={
+        <div>
+          <p>
+            {t('me:apiKeyMessage1')}
+            <br />
+            <br />
+            {t('me:apiKeyMessage3')}
+          </p>
```

Comment:
Avoid using line breaks. Perhaps we can use `<p>` tags instead to separate the text like done in src/features/user/Widget/DonationLink/index.tsx